simulator/carla_autoware/README.md

```diff
@@ -0,0 +1,58 @@
+# CARLA Autoware
+
+# ROS2/Autoware.universe bridge for CARLA simulator
+
+Thanks to <https://github.com/gezp> for ROS2 Humble support for CARLA Communication.
+This ros package enables communication between Autoware and CARLA for autonomous driving simulation.
+
+# Environment
+
+| ubuntu |  ros   | carla  | autoware |
+| :----: | :----: | :----: | :------: |
+| 22.04  | humble | 0.9.15 | 2024.04  |
+
+# Setup
+
+## install
+
+- [Autoware.Universe](https://autowarefoundation.github.io/autoware-documentation/galactic/installation/autoware/source-installation/)
+- [CARLA Installation](https://carla.readthedocs.io/en/latest/start_quickstart/)
+- [Carla Lanelet2 Maps](https://bitbucket.org/carla-simulator/autoware-contents/src/master/maps/)
+- [Python wheel for CARLA 0.9.15 Ros2 Humble communication](https://github.com/gezp/carla_ros/releases/tag/carla-0.9.15-ubuntu-22.04)
```

Comment:
Can you add some extra details for this step ? I think there are 2 possible ways: install the wheel using some commands (I actually don't know how to do it); or add the egg file to the `PYTHONPATH`.